Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "risc0-verifier"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.60"
+license = "Apache-2.0"
```

How did you compute it?

I used `cargo-msrv` and that say `1.73.0`


```suggestion
rust-version = "1.73"
```

If we define it we should also add a CI step for it:

```yaml
msrv:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v4
    - uses: taiki-e/install-action@cargo-hack
    - run: cargo hack check --rust-version --workspace --ignore-private
```

---

Maybe is better to add it to `Makefile.toml` I'll do some tests.

---

The value `1.60` comes from fflonk verifier library, let's remove it for now.